admin/spec/controllers/check-form.spec.js

```diff
@@ -139,17 +139,17 @@ describe('check-form controller:', () => {
 
     describe('Saving a form', () => {
       beforeEach(() => {
-        const checkForm = new CheckForm
-        checkForm.validate = () => {}
-        checkForm.save = () => {}
-
         spyOn(checkFormService, 'populateFromFile').and.returnValue(checkFormMock)
         spyOn(CheckForm, 'create').and.returnValue(Promise.resolve(checkFormMock))
         spyOn(fs, 'remove').and.returnValue(checkFormMock)
         controller = proxyquire('../../controllers/check-form', {}).saveCheckForm
       })
 
       it('should save the form and redirect the user', async (done) => {
+        const checkForm = new CheckForm()
```

Comment:
If this has been refactored we should not have mongoose models in controllers.  They should be strictly in the data-access layer.

Reply:
yes.  we are about to start the data layer refactoring and it will be much harder if there are mongoose dependencies in controllers and services